### PR TITLE
Update docs regarding base env use

### PR DESCRIPTION
**Pull request by DaveKaretnyk**
### Update conda-build docs to recommend use with base-env only
Documentation solution to problem discussed here: https://github.com/conda/conda-build/issues/4995

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [x] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

**Comment by kenodegard** [comment]
@conda-bot check

File: docs/source/install-conda-build.rst [hunk 1]
```diff
@@ -9,6 +9,40 @@ To enable building conda packages:
...
+.. _-conda-build-wow:
+
+Way of working
+==============
+For proper functioning, it is strongly recommended to install conda-build in
+the conda ``base`` environment. Not doing so may lead to problems.
+
+**Explanation**
+
+With earlier conda / conda-build versions it was possible to build packages in
```
Comment (kathatherine) [inline docs/source/install-conda-build.rst:21]:
```suggestion
With earlier conda / conda-build versions, it was possible to build packages in
```

File: docs/source/install-conda-build.rst [hunk 2]
```diff
@@ -9,6 +9,40 @@ To enable building conda packages:
...
+your own defined environment, e.g. ``my_build_env``. This was partly driven by
+the general conda recommendation not to use the ``base`` env for normal work,
```
Comment (kathatherine) [inline docs/source/install-conda-build.rst:23]:
```suggestion
the general conda recommendation not to use the ``base`` env for normal work;
```

File: docs/source/install-conda-build.rst [hunk 3]
```diff
@@ -9,6 +9,40 @@ To enable building conda packages:
...
+see `Conda Managing Environments`_ for instance. However conda-build is better
+viewed as part of the conda infrastructure, and not as a normal package. Hence
```
Comment (kathatherine) [inline docs/source/install-conda-build.rst:25]:
```suggestion
viewed as part of the conda infrastructure, and not as a normal package. Hence,
```

File: docs/source/install-conda-build.rst [hunk 4]
```diff
@@ -9,6 +9,40 @@ To enable building conda packages:
...
+installing it in the ``base`` env makes more sense. More information:
+`Must conda-build be installed in the base envt?`_
+
+**Other considerations**
+
+* Installing or updating conda-build (in fact any package) in the ``base``
+  environment needs to be run from an account with the proper permissions.
+  I.e. the same permissions as were used to install conda and the base env in
+  the first place via the Miniconda or Anaconda installers. E.g. on Windows
+  that might mean an account with administrator privileges.
```
Comment (kathatherine) [inline docs/source/install-conda-build.rst:35]:
```suggestion
* An installation or update of conda-build (in fact, of any package) in the ``base``
  environment needs to be run from an account with the proper permissions
  (i.e., the same permissions as were used to install conda and the base env in
  the first place via the Miniconda or Anaconda installers). For example, on Windows
  that might mean an account with administrator privileges.
```

File: docs/source/install-conda-build.rst [hunk 5]
```diff
@@ -9,6 +9,40 @@ To enable building conda packages:
...
+
+* `conda-verfiy`_ can be usefully added to the base environment also. This
+  will remove some warnings generated when conda-build runs. And the package
+  is useful in its own right anyway.
```
Comment (kathatherine) [inline docs/source/install-conda-build.rst:39]:
```suggestion
* `conda-verfiy`_ is a useful package that can also be added to the base environment in order
  to remove some warnings generated when conda-build runs.
```
That last sentence is fine if you want keep it. I just figured it was redundant to saying "useful/usefully" in the first sentence.

File: docs/source/install-conda-build.rst [hunk 6]
```diff
@@ -9,6 +9,40 @@ To enable building conda packages:
...
+
+* For critical CI/CD projects, you might want to pin to an explicit (but recent)
+  version of conda-build. I.e. only update to a newer version of conda-build,
+  and conda, once they have been first verified 'offline'.
```
Comment (kathatherine) [inline docs/source/install-conda-build.rst:43]:
```suggestion
* For critical CI/CD projects, you might want to pin to an explicit (but recent)
  version of conda-build, i.e. only update to a newer version of conda-build,
  and conda once they have been first verified 'offline'.
```

**Review by kathatherine — CHANGES_REQUESTED** [review]
Got some general punctuation/wording suggestions. Let me know if you have any questions and thanks for your contribution!

File: docs/source/install-conda-build.rst (hunk 2 above)
Comment (DaveKaretnyk) [reply docs/source/install-conda-build.rst]: ok

File: docs/source/install-conda-build.rst (hunk 4 above)
Comment (DaveKaretnyk) [reply docs/source/install-conda-build.rst]: ok

File: docs/source/install-conda-build.rst (hunk 5 above)
Comment (DaveKaretnyk) [reply docs/source/install-conda-build.rst]: ok

File: docs/source/install-conda-build.rst (hunk 6 above)
Comment (DaveKaretnyk) [reply docs/source/install-conda-build.rst]: ok

**Comment by DaveKaretnyk** [comment]
@kathatherine: thanks, updated content as per house style.

**Review by kathatherine — DISMISSED** [review]
Thanks!

File: docs/source/install-conda-build.rst [hunk 7]
```diff
@@ -9,6 +9,39 @@ To enable building conda packages:
...
+==============
+For proper functioning, it is strongly recommended to install conda-build in
+the conda ``base`` environment. Not doing so may lead to problems.
+
+**Explanation**
+
+With earlier conda / conda-build versions, it was possible to build packages in
+your own defined environment, e.g. ``my_build_env``. This was partly driven by
+the general conda recommendation not to use the ``base`` env for normal work;
+see `Conda Managing Environments`_ for instance. However conda-build is better
```
Comment (beeankha) [inline docs/source/install-conda-build.rst:24]:
```suggestion
see `Conda Managing Environments`_ for instance. However, conda-build is better
```

File: docs/source/install-conda-build.rst [hunk 8]
```diff
@@ -9,6 +9,39 @@ To enable building conda packages:
...
+viewed as part of the conda infrastructure, and not as a normal package. Hence,
+installing it in the ``base`` env makes more sense. More information:
+`Must conda-build be installed in the base envt?`_
+
+**Other considerations**
+
+* An installiion or update of conda-build (in fact, of any package) in the ``base``
```
Comment (beeankha) [inline docs/source/install-conda-build.rst:31]:
```suggestion
* An installation or update of conda-build (in fact, of any package) in the ``base``
```

File: docs/source/install-conda-build.rst [hunk 9]
```diff
@@ -9,6 +9,39 @@ To enable building conda packages:
...
+  (i.e., the same permissions as were used to install conda and the base env in
+  the first place via the Miniconda or Anaconda installers). For example, on
+  Windows that might mean an account with administrator privileges.
+
+* `conda-verfiy`_ is a useful package that can also be added to the base
+  environment in order to remove some warnings generated when conda-build runs.
+
+* For critical CI/CD projects, you might want to pin to an explicit (but recent)
+  version of conda-build, i.e. only update to a newer version of conda-build,
+  and conda, once they have been first verified 'offline'.
```
Comment (beeankha) [inline docs/source/install-conda-build.rst:42]:
```suggestion
* For critical CI/CD projects, you might want to pin to an explicit (but recent)
  version of conda-build, i.e. only update to a newer version of conda-build
  and conda once they have been first verified "offline".
```

**Review by beeankha — DISMISSED** [review]
A few edit suggestions, otherwise LGTM!

File: docs/source/install-conda-build.rst [hunk 10]
```diff
@@ -9,6 +9,39 @@ To enable building conda packages:
 * update conda and conda-build
 
 
+.. _-conda-build-wow:
+
+Way of working
+==============
+For proper functioning, it is strongly recommended to install conda-build in
```
Comment (jezdez) [inline docs/source/install-conda-build.rst:16]:
```suggestion

For proper functioning, it is strongly recommended to install conda-build in
```

File: docs/source/install-conda-build.rst [hunk 11]
```diff
@@ -9,6 +9,40 @@ To enable building conda packages:
...
 
+.. _-conda-build-wow:
+
+Way of working
+==============
+
+For proper functioning, it is strongly recommended to install conda-build in
+the conda ``base`` environment. Not doing so may lead to problems.
+
+**Explanation**
```
Comment (jezdez) [inline docs/source/install-conda-build.rst:20]:
```suggestion
Explanation
-----------
```

File: docs/source/install-conda-build.rst [hunk 12]
```diff
@@ -9,6 +9,40 @@ To enable building conda packages:
...
+
+With earlier conda / conda-build versions, it was possible to build packages in
+your own defined environment, e.g. ``my_build_env``. This was partly driven by
+the general conda recommendation not to use the ``base`` env for normal work;
+see `Conda Managing Environments`_ for instance. However, conda-build is better
+viewed as part of the conda infrastructure, and not as a normal package. Hence,
+installing it in the ``base`` env makes more sense. More information:
+`Must conda-build be installed in the base envt?`_
+
+**Other considerations**
```
Comment (jezdez) [inline docs/source/install-conda-build.rst:30]:
```suggestion
Other considerations
--------------------
```

File: news/4995-run-conda-build-in-base-env [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+
+* <news item>
+
+### Deprecations
+
+* <news item>
+
+### Docs
+
+* conda-build should be run from the base env (#4995)
```
Comment (jezdez) [inline news/4995-run-conda-build-in-base-env:15]:
```suggestion
* Add explanation that conda-build should be run from the base env. (#4995)
```

**Review by jezdez — APPROVED** [review]

**Comment by DaveKaretnyk** [comment]
Thanks all for the review comments, making the final changes, and merging.

I have a question regarding your procedures, especially for doc changes like this. Should the submitter (me)  make / discuss the review suggestions or should it be left to Anaconda employees to take care of? E.g. a bunch of them are house style changes....

**Comment by kenodegard** [comment]
@DaveKaretnyk either works, simple grammatical and formatting changes are often just committed by maintainers (done with good intentions in an effort to decrease the amount of back and forth necessary), more substantial changes will likely to be discussed further

**Pull request merged**